chrome/background.js

```diff
@@ -21,9 +24,36 @@ function fillLoginForm(login) {
       });
     }
   );
+
+  if (login.digits) {
+    tabInfos[tab.id] = {
+      login: loginParam,
+      hostname: getHostname(tab.url)
+    };
+    displayOTP(tab.id);
+  }
+}
+
+function displayOTP(tabId) {
+  chrome.tabs.executeScript(
+    tabId,
+    {
+      file: "/inject_otp.js"
+    },
+    function() {
+      chrome.tabs.executeScript(
+        tabId,
+        {
+          code: `browserpassDisplayOTP(${tabInfos[tabId].login});`
+        }
+      );
+    }
+  );
 }
 
 function onMessage(request, sender, sendResponse) {
+  console.log(JSON.stringify(request));
```

Comment:
need to remember to delete this before merging

Reply:
Oops lol, thanks for reminding me :)